.circleci/config.yml

```diff
@@ -196,12 +196,14 @@ jobs:
                 command: |
                   source scripts/utils.sh
                   tag=v$(extract_version_from_current_wd)
-                  # Newlines (<br/>) are necessary for better layout on Marketplace, but make the formatting worse on GitHub
-                  change_notes=$(sed '/^<br.>$/d' CHANGE-NOTES.html)
-                  hub release create \
+                  change_notes=$(./gradlew -q getReleaseChangeLogForGithub)
```

Comment:
Hmm this will be automatically derived from commit messages? 🤔 note that we only ever like to put user-facing changes in the changelog; technical debt tasks etc. shouldn't be included coz then the change notes will have close to zero value for the user

Reply:
I'd rather expect the developers themselves to fill up these change notes manually, as is currently; these notes are hard to generate automatically

Reply:
(ok, seems resolved by now)